server/etcdmain/config.go

```diff
@@ -276,6 +276,10 @@ func newConfig() *config {
 	// gateway
 	fs.BoolVar(&cfg.ec.EnableGRPCGateway, "enable-grpc-gateway", cfg.ec.EnableGRPCGateway, "Enable GRPC gateway.")
 
+	// grpc dump
+	fs.BoolVar(&cfg.ec.EnableGRPCDump, "enable-grpc-dump", false, "Enable GRPC requests & response dump.")
```

Comment:
I would also be interested in having these be also part of `etcdutl` or `etcdctl`, as for example if I don't want to interrupt etcd that is managed I would like to send a signal to say "start grpc dump" from the CLI tools. What do you think?

Reply:
let me take a look how to integrate this in `etcdctl` or `etcdutl`

Reply:
I think it belongs to `etcdctl`, especially if the Start & Stop calls are part of public etcd API (clientv3 surface).